src/main/java/sparksoniq/jsoniq/runtime/iterator/functions/base/Functions.java

```diff
@@ -123,8 +139,109 @@
 import java.util.HashMap;
 import java.util.List;
 
-
-import static sparksoniq.jsoniq.runtime.iterator.functions.base.Functions.FunctionNames.*;
+import static sparksoniq.jsoniq.runtime.iterator.functions.base.Functions.FunctionNames.ABS;
```

Comment:
Is this intentional? I saw this change go back and forth. I don't mind either way, but it would be nice to settle it.

Reply:
Nice catch! I configured in IntelliJ some time ago to not use wildcard imports. I just checked a PR of @AndreaRinaldi1 which I assume to be using the default import styling of IntelliJ which enables wildcard imports. 

No wildcard imports prevent unexpected collisions in the namespace, however this can be regarded as a somewhat rare case. Wildcard imports reduce the number of lines in our imports significantly. Which may or may not be regarded as important for readability. 

I have no strong preferences over either one. What do you think @AndreaRinaldi1 ?

Reply:
Yes I think I had also configured IntelliJ to use single imports but probably we surpassed the count that I previously set for '*' imports. I just increased the counter to 999. 
We can use single imports, if we prefer.